apps/dotcom/src/app/pages/contact/Contact.tsx

```diff
@@ -0,0 +1,145 @@
+  background: "linear-gradient(127deg, rgba(12,0,213,1) 0%, rgba(248,0,255,1) 100%)" 
+});
+
+const Contact: React.FC = () => {
+
+  interface ContactData {
+    firstName: string;
+    lastName: string;
+    email: string;
+    year: string;
+    major: string;
+    // TODO: ask if we still want to do user-submitted questions
+    questions: any[];
+    role: string;
+  }
+
+  const [data, setData] = useState({ firstName: '', lastName: '', email: '', year: '', major: '', questions: [], role: '' } as ContactData);
+  const [showAlert, setShowAlert] = useState<boolean>(false)
+
+  return (
+    <div>
+      <Container maxWidth="md">
+        <Helmet>
+          <title>Projects</title>
+          <meta
+            name="description"
+            content="C4C delivers web applications to Boston-based nonprofits."
+          />
+        </Helmet>
+        <Hero
+          subtitle="Want to stay in touch? Fill out your contact info below!"
+          title="Contact Us"
+          SvgNode={ContactSvg}
+        />
+      </Container>
+      <Container maxWidth="md">
+        <Box marginTop={5}>
+          <Grid container>
+            <TextField 
+              label="First Name"
+              value={data.firstName}
+              onChange={(event) => setData({...data, firstName: event.target.value})}
+            />
+            <TextField 
+              label="Last Name"
+              value={data.lastName}
+              onChange={(event) => setData({...data, lastName: event.target.value})}
+            />
+            <TextField 
+              label="Email"
+              value={data.email}
+              onChange={(event) => setData({...data, email: event.target.value})}
+            />
+            <TextField 
+              label="Graduation Year"
+              value={data.year}
+              onChange={(event) => setData({...data, year: event.target.value})}
+            />
+            <TextField 
+              label="Major"
+              value={data.major}
+              onChange={(event) => setData({...data, major: event.target.value})}
+            />
+          </Grid>
+          <StyledDivider />
+          <FormLabel component="legend">Which positions are you interested in?</FormLabel>        
+          <RadioGroup value={data.role} row>
+            <FormControlLabel value="developer" control={<Radio />} label="Developer" 
+              onChange={(event: any) => setData({...data, role: "developer"})}
+            />
+            <FormControlLabel value="designer" control={<Radio />} label="Designer"
+              onChange={(event: any) => setData({...data, role: "designer"})}
+            />
+            <FormControlLabel value="productmanager" control={<Radio />} label="Product Manager"
+              onChange={(event: any) => setData({...data, role: "productmanager"})}
+            />
+          </RadioGroup>
+          <StyledButton onClick={(event) => {
+            console.log('submitting form', data);
```

Comment:
This is a pretty big function - could we possibly break it  up into a few smaller functions?